src/sig/wasm.rs

```diff
@@ -0,0 +1,300 @@
+use wasm_bindgen::prelude::*;
+
+use rand_chacha::ChaChaRng;
+use rand_core::{RngCore, SeedableRng};
+
+use crate::{
+    curve::zexe::PairingCurve as Bls12_377,
```

Comment:
I would really much prefer to keep the signatures algorithm, the curves they use and some specific compiling (wasm) separated as much as possible. Here it introduces both wasm and zexe curve in the sig package.
Would it be possible to make a `threshold::wasm` package ? that can also compiles using other curves ? Like can it be made with the generic traits `Curve` `Point` `Scalar` ?

Reply:
Wasm support is not really generic friendly so we'd have to concretely instantiate it for each curve. I will move it to a separate module though, np.